discovery-provider/alembic/trigger_sql/handle_playlist.sql

```diff
@@ -30,6 +33,30 @@ begin
     where user_id = new.playlist_owner_id;
   end if;
 
+  begin
+    for track_item IN select jsonb_array_elements from jsonb_array_elements(new.playlist_contents -> 'track_ids')
+    loop
+      if (track_item->>'time')::double precision::int >= extract(epoch from new.updated_at)::int then
```

Comment:
it seems unlikely this if condition would work in practice

the `time` is set on the client when a track is added to playlist... but isn't the `updated_at` is set on the server? (I think to the block time, but maybe to the system clock?  I'm not sure).

Either way it seems like the `time` will always be before `updated_at`, right?

Reply:
Actually I think I was discussion with isaac that the client does set `time` but maybe the server re-writes it on processing... in which case it might be possible that server sets `updated_at` and `time` to the same value (the blocktime)... in which case this would be okay.

Either way... I think in general another option would be to take the `old` and `new` row in the trigger and do a set difference to find IDs in new that are not in old.

Reply:
the server re-writes the time during indexing so it should work correctly and in fact be exactly the same. 
although fetching the old row and diff-ing it would also be good too